src/ig_media_filter/function.json

```diff
@@ -0,0 +1,29 @@
+{
+  "bindings": [
+    {
+      "name": "msg",
+      "type": "queueTrigger",
+      "direction": "in",
+      "queueName": "ig-subscription-media",
+      "connection": "AzureWebJobsStorage"
+    },
+    {
+      "name": "token",
+      "type": "documentDB",
+      "direction": "in",
+      "connection": "DocDb_ConnectionString",
+      "databaseName": "mean-dev",
+      "collectionName": "users",
+      "sqlQuery": "SELECT c.instagram.token FROM c WHERE c.instagram['$id'] = {user_id}"
```

Comment:
this query searches the docdb store for registered users and returns the token if it exists